internal/wrappers/results-summary.go

```diff
@@ -235,24 +243,27 @@ const summaryTemplateHeader = `{{define "SummaryTemplate"}}
         }
 
         .bg-kicks {
-            background-color: #008e96 !important;
+            background-color: #079E9E !important;
         }
 
         .bg-red {
             background-color: #f1605d;
         }
 
         .bg-sast {
-            background-color: #1165b4 !important;
+            background-color: #0356A5 !important;
         }
 
         .bg-sca {
-            background-color: #0fcdc2 !important;
+            background-color: #15D7D7 !important;
```

Comment:
what are these color changes?

Reply:
I change the colors of the HTML report to be exactly like in the UI Figma